docs/maintainers/maintainers-guide.md

```diff
@@ -70,39 +70,39 @@ There are two ways to add contributors to a Contributor Insight Page:
 1. Create a new Contributor Insight Page with selected contributors.
 2. Add selected contributors to an existing Contributor Insight Page.
 
-## Contributor Insights : Connecting with Contributors
+## Contributor Insights: Connecting with Contributors
 
 OpenSauced Contributor Insights feature allows you to categorize, monitor, and analyze various groups of contributors. You can use Contributor Insights to track the contributions of a specific team, a specific repository, or a group of repositories. This feature offers granular insights into each contributor's activities and contributions, helping maintainers to track individual contribution histories, identify active engagement, recognize new or alumni contributors, and compare the performance of different contributors.
 
 The Contributor Insights features allow you to:
 
 - track the commits and PRs of the contributors on your list,
 - filter your contributors by their activity level, including most active, new, and alumni,
-- view the repositories that your contributors have contributed to, the top programming languages they've used, and their time zones.
+- view the repositories your contributors have contributed to, the top programming languages they've used, and their time zones.
 
 ### Why Create a Contributor Insight Page?
 
 There are many reasons to create a Contributor Insight Page, but here's some inspiration to get you started:
 
 - **Interacting contributors**: Keep tabs on contributors actively interacting with your repository.
 - **Organization's developers**: Monitor the contributions and performance of your internal team.
-- **VIP contributors**: Highlight and track contributors who you've identifies as key maintainers or significant contributors.
+- **VIP contributors**: Highlight and track contributors you've identified as key maintainers or significant contributors.
 - **Sales leads**: Identify contributors whose engagement could potentially be turned into sales opportunities.
 - **Potential hires**: Observe contributors who exhibit skill and commitment, making them prime candidates for hiring.
 - **Alumni contributors**: Keep an eye on contributors who have decreased their engagement or left the project.
 
 ### Creating a New Contributor Insight Page
 
-1. Click the + next to Insights in the sidebar to create a new insight page. You will be redirected to a page where you can create your new insight page.
+1. Click the "+" next to "Insights" in the sidebar.
+2. Click "New Contributor Insight". You will be redirected to a page where you can create your new Contributor Insight Page.
```

Comment:
```suggestion
2. Click "New Contributor Insight". From there, you will be redirected to a page where you can create your new Contributor Insight Page.
```

Reply:
@CBID2 punctuation should go inside the quotes. I also don't feel this updates are necessary.